src/commander/commander.ts

```diff
@@ -62,13 +63,16 @@ export function getCommanderFromConfiguration(
       parsedCommand = new CancelCommand(taskId || "");
     });
 
-  for (const ignoredCommand of botPullRequestIgnoreCommands) {
-    root
-      .command(ignoredCommand)
-      .exitOverride()
-      .action(() => {
-        parsedCommand = new SkipEvent(`Ignored command: ${ignoredCommand}`);
-      });
+  // ignore `bot merge` / `bot rebase` for repos which support processbot
+  if (processBotSupportedRepos.includes(repo)) {
+    for (const ignoredCommand of botPullRequestIgnoreCommands) {
```

Comment:
Would it make sense to rename `botPullRequestIgnoreCommands` to something like `processBotIgnoreCommands`?

Since those are commands that are ignored only in the context of the processbot?

Reply:
yes, that makes sense, but i hope we will remove them soonish, once move to polkadot-sdk. There won't be ignored commands and processbot, it's more temporary plug